recipes/feather-format/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]; then
+    # Building Feather requires a C++11 compiler, which requires OS X 10.9+
+    export MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
10.10 seems very restrictive.  I used 10.9 when I built the feather-format 0.1 package in my channel and have not received any emails about it not working.  

I submitted a build using 10.7, lets see if that works.  From what I recall, 10.9 was the earliest I could set the target and have the package build, but my memory of that is a bit hazy and I may have been missing the correct SDKs on my machine.